### PR TITLE
chore: update control-plane dependency

### PR DESCRIPTION
**Pull request by tiagolobocastro**
This brings the update to replica-topology.
Also update README file.

### CODE REVIEW

File: k8s/plugin/README.md [hunk 1]
```diff
@@ -28,31 +28,31 @@ The plugin needs to be able to connect to the REST server in order to make the a
 1. Get Volumes
 ```
 ❯ kubectl mayastor get volumes
```
Comment (niladrih) [inline k8s/plugin/README.md:30]: I think we should have namespaces on these sample commands. It is unintuitive to have the default namespace set to 'mayastor'.

File: k8s/plugin/README.md [hunk 2]
```diff
@@ -28,31 +28,31 @@ The plugin needs to be able to connect to the REST server in order to make the a
...
 ```
 ❯ kubectl mayastor get pools
- ID               TOTAL CAPACITY  USED CAPACITY  DISKS                                                     NODE      STATUS  MANAGED
- mayastor-pool-1  5360320512      1111490560     aio:///dev/vdb?uuid=d8a36b4b-0435-4fee-bf76-f2aef980b833  kworker1  Online  true
- mayastor-pool-2  5360320512      2172649472     aio:///dev/vdc?uuid=bb12ec7d-8fc3-4644-82cd-dee5b63fc8c5  kworker1  Online  true
- mayastor-pool-3  5360320512      3258974208     aio:///dev/vdb?uuid=f324edb7-1aca-41ec-954a-9614527f77e1  kworker2  Online  false
+ ID               DISKS                                                     MANAGED  NODE      STATUS  CAPACITY  ALLOCATED  AVAILABLE
+ mayastor-pool-1  aio:///dev/vdb?uuid=d8a36b4b-0435-4fee-bf76-f2aef980b833  true     kworker1  Online  500GiB    100GiB     400GiB
+ mayastor-pool-2  aio:///dev/vdc?uuid=bb12ec7d-8fc3-4644-82cd-dee5b63fc8c5  true     kworker1  Online  500GiB    100GiB     400GiB
+ mayastor-pool-3  aio:///dev/vdb?uuid=f324edb7-1aca-41ec-954a-9614527f77e1  true     kworker2  Online  500GiB    100GiB     400GiB
```
Comment (niladrih) [inline k8s/plugin/README.md:49]:
```suggestion
 mayastor-pool-kworker1  aio:///dev/vdb?uuid=d8a36b4b-0435-4fee-bf76-f2aef980b833  true     kworker1  Online  500GiB    100GiB     400GiB
 mayastor-pool-kworker2  aio:///dev/vdc?uuid=bb12ec7d-8fc3-4644-82cd-dee5b63fc8c5  true     kworker1  Online  500GiB    100GiB     400GiB
 mayastor-pool-kworker3  aio:///dev/vdb?uuid=f324edb7-1aca-41ec-954a-9614527f77e1  true     kworker2  Online  500GiB    100GiB     400GiB
```

**Review by niladrih — APPROVED** [review]
Nitpick comments.

File: k8s/plugin/README.md (hunk 2 above)
Comment (tiagolobocastro) [reply k8s/plugin/README.md:49]: Done

File: k8s/plugin/README.md (hunk 1 above)
Comment (tiagolobocastro) [reply k8s/plugin/README.md:64]: I've updated README to detail what the default namespace is.

File: k8s/plugin/README.md [hunk 3]
```diff
@@ -28,39 +62,39 @@ The plugin needs to be able to connect to the REST server in order to make the a
...
- mayastor-pool-2  5360320512      2172649472     aio:///dev/vdc?uuid=bb12ec7d-8fc3-4644-82cd-dee5b63fc8c5  kworker1  Online  true
- mayastor-pool-3  5360320512      3258974208     aio:///dev/vdb?uuid=f324edb7-1aca-41ec-954a-9614527f77e1  kworker2  Online  false
+ ID               DISKS                                                     MANAGED  NODE      STATUS  CAPACITY  ALLOCATED  AVAILABLE
+ pool-1-kworker1  aio:///dev/vdb?uuid=d8a36b4b-0435-4fee-bf76-f2aef980b833  true     kworker1  Online  500GiB    100GiB     400GiB
+ pool-1-kworker2  aio:///dev/vdc?uuid=bb12ec7d-8fc3-4644-82cd-dee5b63fc8c5  true     kworker1  Online  500GiB    100GiB     400GiB
+ pool-1-kworker2  aio:///dev/vdb?uuid=f324edb7-1aca-41ec-954a-9614527f77e1  true     kworker2  Online  500GiB    100GiB     400GiB
 ```
 4. Get Pool by ID
 ```
 ❯ kubectl mayastor get pool mayastor-pool-1
```
Comment (niladrih) [inline k8s/plugin/README.md:87]: This one...

**Review by niladrih — COMMENTED** [review]
Nitpick comment.

File: k8s/plugin/README.md (hunk 3 above)
Comment (tiagolobocastro) [reply k8s/plugin/README.md:87]: I added the help on top where it shows the default namespace is mayastor, so don't need to keep repeating it.

Comment (niladrih) [reply k8s/plugin/README.md:87]: No, I mean the pool name. It's pool-1-kworker1

**Review by Abhinandan-Purkait — APPROVED** [review]

File: k8s/plugin/README.md (hunk 3 above)
Comment (tiagolobocastro) [reply k8s/plugin/README.md]: Oh good catch, think that's all now

**Pull request merged**